---
camera: 
  -
    ns: bfly
    serial: 21137010
    mode: auto
    alias: cam0
    sub_ns: cam0
    publish_all_images: true
    expo_t_major_lower: 1000.0 
    expo_t_major_upper: 20000.0
    gain_major_lower: 0.0
    gain_major_upper: 20.0
    expo_val_offset: 200.0
...